src/box/tuple.h

```diff
@@ -1449,6 +1449,18 @@ tuple_to_buf(struct tuple *tuple, char *buf, size_t size);
 size_t
 tuple_runtime_memory_used(void);
 
+/**
+ * Allocate size bytes on runtime arena for tuples.
+ */
+void *
+tuple_runtime_memory_alloc(size_t size);
+
+/**
+ * Free memory of size bytes allocated on runtime arena for tuples.
+ */
+void
+tuple_runtime_memory_free(void *ptr, size_t size);
```

Comment:
Discussed it with @alyapunov, we decided not to store size in a header in order to preserve this feature of the allocator and to avoid alignment problems.